exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py

```diff
@@ -28,25 +30,73 @@ def arg_parser():
         "--jupyter-server", metavar="<PATH-TO-JUPYTER-BINARY>",
         help="start server for Jupyter notebooks",
     )
+    parser.add_argument(
+        "--user", type=str,
+        help="user name for running jupyter server",
+    )
+    parser.add_argument(
+        "--jupyter-logfile", type=Path,
+        help="path to write Jupyter server log messages to",
+    )
     parser.add_argument(
         "--warning-as-error", action="store_true",
         help="treat warning as error",
     )
     return parser
 
 
-def start_jupyter_server(binary_path: str, notebook_dir: str):
-    subprocess.run([
+def start_jupyter_server(
+        binary_path: str,
+        notebook_dir: str,
+        logfile: Path,
+        user: str,
+        poll_sleep: float = 1,
+):
+    """
+    :param poll_sleep: specifies the waiting time in seconds before reading
+    the a line from the logfile.
+    """
+    def exit_on_error(rc):
+        if rc is not None and rc != 0:
+            log_messages = logfile.read_text()
+            print(
+                f"Jupyter Server terminated with error code {rc},"
+                f" Logfile {logfile} contains:\n{log_messages}",
+                flush=True,
+            )
+            sys.exit(rc)
+
+    command_line = [
         binary_path,
         f"--notebook-dir={notebook_dir}",
         "--no-browser",
         "--allow-root",
-    ])
-    print(cleandoc(
-        """
-        Server for Jupyter has been started.
-        You can connect with https://localhost:8888
-        """))
+    ]
+    with open(logfile, "w") as f:
+        p = subprocess.Popen(command_line, stdout=f, stderr=f)
+
+    success_message = cleandoc(f"""
+        Server for Jupyter has been started successfully.
+        You can connect with https://<ip address>:8888.
+        If using a Docker daemon on your local machine then ip address is localhost.
+
+        ┬ ┬┌─┐┌┬┐┌─┐┌┬┐┌─┐  ┬ ┬┌─┐┬ ┬┬─┐   ┬┬ ┬┌─┐┬ ┬┌┬┐┌─┐┬─┐  ┌─┐┌─┐┌─┐┌─┐┬ ┬┌─┐┬─┐┌┬┐ ┬
+        │ │├─┘ ││├─┤ │ ├┤   └┬┘│ ││ │├┬┘   ││ │├─┘└┬┘ │ ├┤ ├┬┘  ├─┘├─┤└─┐└─┐││││ │├┬┘ ││ │
+        └─┘┴  ─┴┘┴ ┴ ┴ └─┘   ┴ └─┘└─┘┴└─  └┘└─┘┴   ┴  ┴ └─┘┴└─  ┴  ┴ ┴└─┘└─┘└┴┘└─┘┴└──┴┘ o
+
+        To update the password as user {user} run
+            {binary_path} server <new password>
```

Comment:
```suggestion
            {binary_path} server password
```

Reply:
see here
https://github.com/exasol/data-science-sandbox/blob/83ed33310b8e8945c351f32c5675317a1eb5b234/exasol/ds/sandbox/runtime/ansible/roles/jupyter/templates/etc/update-motd.d/999-jupyter#L6

Reply:
also we should show the default password here

Reply:
I feared you would say this.
No prop - I think I can add it. :slightly_smiling_face: